packages/axios-extension/src/abap/ui5-abap-repository-service.ts

```diff
@@ -204,19 +206,25 @@ export class Ui5AbapRepositoryService extends ODataService {
         const config = this.createConfig(bsp.transport, testMode);
         const host = this.getAbapFrontendUrl();
         const info: AppInfo = await this.getInfo(bsp.name);
+        const isDest = /\.dest\//.test(config.baseURL);
```

Comment:
Is there any other usage for the `isDest` flag except the usage when logging? If not, why don't we do the check in `prettyPrintMessage`? That would simplify the signature of the function and would remove the usage of the variable in multiple places.

Reply:
Yes, its only used for logging but the issue is, the host being passed down to the print methods is passing the public URL rather than the BAS destination URL.

Reply:
ok